content/solana-nft.md

```diff
@@ -0,0 +1,567 @@
+![Screenshot of Metadata](../assets/solana-nft-metaplex-metadata.png)
+
+### Assets
+
+Before creating a collection you must prepare the assets for your collection. The assets are the images and metadata associated with each NFT. An NFT is created using a pairing of an image and a file containing its metadata.
+
+The metadata should include the following attributes:
+
+- `name` the name of the NFT that will display on-chain
+- `symbol` the optional symbol of the NFT that will display on-chain
+- `description` the description of the NFT
+- `seller_fee_basis_points` the fee collected upon sale of the NFT which is split between the creators
+- `image` the file name of the corresponding image that the NFT will display
+- `attributes` the attributes of the NFT
+- `properties` the list of creators, the creator's share of seller fee basis points, and the file of the image. If there are multiple creators, the total “share” must add to 100
+- `collection` the “name” and “family” of the NFT collection
+
+The metadata file will look something like the following:
+
+```json
+{
+    "name": "Number #0001",
+    "symbol": "NB",
+    "description": "Collection of 10 numbers on the blockchain. This is the number 1/10.",
+    "seller_fee_basis_points": 500,
+    "image": "0.png",
+    "attributes": [
+        {"trait_type": "Layer-1", "value": "0"},
+        {"trait_type": "Layer-2", "value": "0"},
+        {"trait_type": "Layer-3", "value": "0"},
+        {"trait_type": "Layer-4", "value": "1"}
+    ],
+    "properties": {
+        "creators": [{"address": "N4f6zftYsuu4yT7icsjLwh4i6pB1zvvKbseHj2NmSQw", "share": 100}],
+        "files": [{"uri": "0.png", "type": "image/png"}]
+    },
+    "collection": {"name": "numbers", "family": "numbers"}
+}
+```
+
+## Candy Machine v2
+
+Candy Machine v2 is an NFT distribution program created by Metaplex. A candy machine is configured using a JSON file that can be reused. In other words, a creator can use the same configuration file to create each multiple NFT collections. When a new candy machine is created, the configurations are stored in an account on-chain where the configuration fields can be updated using the Candy Machine CLI. You can read more about Candy Machine configurations [here](https://docs.metaplex.com/candy-machine-v2/configuration).
+
+ - `price` is the amount to charge for each NFT minted from the Candy Machine.
+
+ - `number` is the amount of NFTs in the Candy Machine and must match the number of asset pairings you’ve created for your collection.
+
+ - `gatekeeper` enables CAPTCHA verification before minting from the Candy Machine. The address of the currently supported provider network can be found on the Metaplex documentation [here](https://docs.metaplex.com/candy-machine-v2/configuration).  Note that a gatekeeper won't work with a whitelist presale enabled.
+
+    ```json
```

Comment:
@ZYJLiu @mixelpixel can one of you go through all of the code snippets and make sure they don't start indented like this? They should start lined up with the code block and then have a tab for indentation levels beyond that.

Reply:
@jamesrp13 can do - I indented the code section's so it would be "underneath" the relevant bullet-point, just confirming - yoou prefer them to be flush with the parent point?